README.md

```diff
@@ -65,32 +114,23 @@ const notJsonFiles = FileHound
 ##### Returns
 Returns a FileHound instance.
 
-### `.any(FileHound...)  -> Promise`
-
-##### Parameters
-* Accepts one or more instance of FileHound. Will unpack an array.
-
-##### Returns
-Returns a Promise of all matches. If the Promise fulfills, the fulfullment value is an array of all matching files.
-
-### `.findFiles(path, globPattern) -> Promise`
```

Comment:
I've removed `findFiles`. I guess it's a shorthand, but feel like it would be better for the library to present _one_ way to find files - the fluent interface. Adding in another shortcut to find files might make it harder to learn the library. Happy to revert this though if you'd like to keep.